### DISCUSSION TITLE
translate subtitles option

### DISCUSSION THREAD

**Discussion by galactic76** [justdan96/tsMuxer#758] [Enhancements]
While I ask long time ago to add and extraction of subtitles simulatiosly with remuxing the film ( to translate it in third app to other language) may be you can make subtitle translate opion inside your tool.
It will be very nice

cause for example - film every.body.2023 - i cant find any subtitles elswehere, but when i   try to extract it and put in tranlation in third app - it says error ( cause subtitles is pgs i think)

**Comment by Hellboy00000** [comment]
This is no a sub app. I almost 100% sure this is not going to happens. And you are asking for 2 things translation and OCR (convert PGS to SRT). Basically asking for a new app. You need to demux yourself the PGS SUB with tsMuxeR or MKVToolNix, and then do the OCR and then translate.
